addons/sales_team/views/crm_team_views.xml

```diff
@@ -22,12 +20,29 @@
 
         <!-- Case Teams Action -->
         <record id="crm_team_salesteams_act" model="ir.actions.act_window">
+            <field name="name">Sales Teams</field>
+            <field name="res_model">crm.team</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">kanban,form</field>
+            <field name="view_id" ref="crm_team_salesteams_search"/>
+            <field name="context">{'button_name': 'Sales Orders'}</field>
```

Comment:
I did not find where your are using this context key.
Moreover, this is not translatable. So, this is not a good idea to do this. You'd rather pass a technical context key, that will trigger (maybe in the context field handling the label name) the label itself and its translation.